src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -560,7 +579,8 @@ bool CommandRequestImpl::CheckAllowedParameters() {
   smart_objects::SmartMap::const_iterator iter_end = s_map.map_end();
 
   for (; iter != iter_end; ++iter) {
-    if (iter->second.asBool()) {
+    if (smart_objects::SmartType_Null != iter->second.getType() &&
+        smart_objects::SmartType_Invalid != iter->second.getType()) {
```

Comment:
@AKalinich-Luxoft May be we can use here `helpers::Compare` from utils?

Reply:
@AByzhynar fixed in ac2f5e9fa37dab60ed9abd75421f70e3ae72d6fc